habitat-lab/habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -5,17 +5,17 @@
 # LICENSE file in the root directory of this source tree.
 
 import os.path as osp
-from collections import defaultdict
+from collections import Sequence, defaultdict
 from typing import Any, Dict, List, Optional, Tuple
 
 import magnum as mn
 import numpy as np
 from tqdm import tqdm
-from yacs.config import CfgNode as CN
 
 import habitat.datasets.rearrange.samplers as samplers
 import habitat.sims.habitat_simulator.sim_utilities as sutils
 import habitat_sim
+from habitat.config import DictConfig
```

Comment:
Import `DictConfig` from omegaconf and use `TYPE_CHECKING`, because DictConfig is used only for type declaration here. See [this commit](https://github.com/facebookresearch/habitat-lab/pull/1000/commits/0565fdbfdfa7f546ef29014e74c1c1f7eb950094) as an example.

Reply:
isinstance requires a type as second argument. A string will raise an error. Does it not raise an error in your setup and if not, which version of python are you on ?

Reply:
yes, but in this file it is used only as a type annotation